serverless-now/env/prod.json

```diff
@@ -0,0 +1,10 @@
+{
+    "COGNITO_DOMAIN": "uoapool-sandbox.auth.ap-southeast-2.amazoncognito.com",
```

Comment:
Same thing here as `nonprod.json`

Reply:
Ok, will remove domain and the parameter store paths.